library/spdm_responder_lib/libspdm_rsp_psk_exchange.c

```diff
@@ -359,6 +360,10 @@ return_status spdm_get_response_psk_exchange(IN void *context,
 
         spdm_set_session_state(spdm_context, session_id,
                        LIBSPDM_SESSION_STATE_ESTABLISHED);
+        result = init_watchdog(spdm_context->local_context.heartbeat_period);
```

Comment:
why we need special handling here?
Can we move it together with libspdm_rsp_receive_send.c ?